pkgs/tools/system/tre/default.nix

```
@@ -0,0 +1,24 @@
+{ rustPlatform, fetchFromGitHub, stdenv }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tre-${version}";
+  version = "v0.2.2";
```

```suggestion
  version = "0.2.2";
```

Please move the ```v``` in the tag to rev, as noted below.